AGENTS.md

```
@@ -88,3 +618,512 @@ Keep this file up to date when making significant changes to the codebase:
+```bash
+# Run migrations
+openfga migrate --datastore-engine postgres --datastore-uri 'postgres://...'
+```
+
+## Architecture
+
+### High-Level Structure
+
+OpenFGA follows a layered architecture:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)